Functional.AsyncEnumerables/AsyncTaskEnumerator.cs

```diff
@@ -20,12 +19,18 @@ internal AsyncTaskEnumerator(Lazy<IEnumerator<Task<T>>> enumerator)
 
 		public Task<bool> MoveNext()
 		{
-			var enumerator = _enumerator.Value;
+			IEnumerator<Task<T>> enumerator = _enumerator.Value;
```

Comment:
I see the change to use explicit types and to add braces around one line if/while blocks, and I'd prefer these style changes not be made. If this is what the editorconfig I added says to do, then my bad. I didn't really look at it very closely. I wish I could just export my VS settings.

Reply:
This is because I applied automatic IDE formatter which is new in VS2019 community.  https://devblogs.microsoft.com/visualstudio/little-great-things-about-visual-studio-2019/?utm_source=vs_developer_news&utm_medium=referral

Reply:
Perhaps my editorconfig was bad then. I'll have to take a look at it.

Reply:
No this is not from editorconfig, but some more smart rules introduced in VS2019